package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -182,27 +184,54 @@ preload_rke2_images()
 
       $rke2_tmp/bin/rke2 server &> /rke2.log &
 
+      until [ -f "/var/lib/rancher/rke2/bin/containerd" ] && [ -f "/var/lib/rancher/rke2/agent/etc/containerd/config.toml" ]
```

Comment:
The PR generally looks good, and I've finished installation multiple times.
By checking RKE2/k3s code, here the containerd configuration [is generated from a template.](https://github.com/k3s-io/k3s/blob/7af5b16788afe9ce1718d7b75b35eafac7454705/pkg/agent/containerd/containerd.go#L40)
So RKE2 may be killed immediately after this check and leave a non-complete configuration file.
Can we use the `ctr` command to wait until the containerd brought by RKE2 is ready? Somehow similar to the [wait](https://github.com/k3s-io/k3s/blob/7af5b16788afe9ce1718d7b75b35eafac7454705/pkg/agent/containerd/containerd.go#L96) here.

Reply:
Updated, thanks.